src/core_ondisk.md

```diff
@@ -69,6 +69,15 @@ The EROFS superblock is laid out as follows in [`struct erofs_super_block`](http
 | 0x68   |   __u8 | xattr_filter_reserved | Always 0 for reserved use                                                                          |
 | 0x69   |   __u8 | reserved[23]          | Reserved                                                                                           |
 
+The CRC32-C checksum is calculated from the first byte of the Superblock
+(offset `0x400`) to the end of the physical block that contains it, where the
+block size is defined by blkszbits. For block sizes larger than 1024 bytes,
+the first 1 KB of the block is skipped, because the Superblock itself starts
+at offset `0x400` and does not cover the initial padding area of the block.
```

Comment:
It would be better to document why it behaves as this way, because < 0x400 is reversed for the conditional boot sector + MBR, or users could add their own customized header, and sb chksum shouldn't be impacted for such usage.

Reply:
also it would be better to document when the checksum is calculated, it should assume the checksum field is 0.

Reply:
can you revise a new version? I will merge then.

Reply:
Give me some time, today I have lots of things to do in real world...

Reply:
no worries, feel free to complete this in your spare time ;-)